lib/pages/forward/forward_view.dart

```diff
@@ -125,52 +172,64 @@ class _ForwardViewState extends State<ForwardView> {
             icon: Icons.arrow_back,
             onPressed: () {
               Matrix.of(context).shareContent = null;
-              widget.controller.popScreen();
+              if (sendFromRoomId != null) {
+                context.go('/rooms/$sendFromRoomId');
+              }
```

Comment:
Can you explain for this case ?

Reply:
back to room where user start forward stuff